ycmd/completers/typescript/typescript_completer.py

```diff
@@ -47,6 +50,9 @@ class TypeScriptCompleter( Completer ):
   def __init__( self, user_options ):
     super( TypeScriptCompleter, self ).__init__( user_options )
 
+    # Used to prevent concurrent requests to TSServer.
```

Comment:
You also need to explain in the comment _why_ concurrent requests cause problems. You've explained in the PR, but that won't be easily seen by those reading this code months from now.